Add examples in various languages (#78)
eclipse-ankaios/ankaios: doc/docs/reference/control-interface.md

## Length-delimited protobuf message layout

The messages are encoded using the [length-delimited wire type format](https://protobuf.dev/programming-guides/encoding/#length-types) and layout inside the FIFO file according to the following visualization:

![Length-delimited protobuf message layout inside the file](../assets/length-delimited-protobuf-layout.png)

Every protobuf message is prefixed with its byte length telling the reader how much bytes to read to consume the protobuf message.
The byte length has a dynamic length and is encoded as [VARINT](https://protobuf.dev/programming-guides/encoding/#length-types). 

## Control interface examples

> The subfolder `examples` inside the [Ankaios repository](https://github.com/eclipse-ankaios/ankaios) contains example workload applications in various programming languages that are using the control interface. They demonstrate how to easily use the control interface in self-developed workloads. All examples share the same behavior regardless of the programming language and are simplified to focus on the usage of the control interface but they are not optimized for production usage.

Reviewer: ```suggestion
The subfolder `examples` inside the [Ankaios repository](https://github.com/eclipse-ankaios/ankaios) contains example workload applications in various programming languages that are using the control interface. They demonstrate how to easily use the control interface in self-developed workloads. All examples share the same behavior regardless of the programming language and are simplified to focus on the usage of the control interface. Please note that the examples are not are not optimized for production usage.
```

> The following sections describe important parts of the communication with the Ankaios cluster via the control interface written in Rust. The same concepts are also used in all of the example workload applications.

Reviewer: ```suggestion
The following sections showcase in Rust some important parts of the communication with the Ankaios cluster using the control interface. The same concepts are also used in all of the example workload applications.
```
